eZ/Publish/Core/QueryType/BuildIn/SortSpec/SortClauseParser/MapDistanceSortClauseParser.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\QueryType\BuildIn\SortSpec\SortClauseParser;
+
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause\MapLocationDistance;
+use eZ\Publish\Core\QueryType\BuildIn\SortSpec\SortClauseParserInterface;
+use eZ\Publish\Core\QueryType\BuildIn\SortSpec\SortSpecParserInterface;
+use eZ\Publish\Core\QueryType\BuildIn\SortSpec\Token;
+
+/**
+ * Parser for \eZ\Publish\API\Repository\Values\Content\Query\SortClause\MapLocationDistance sort clause.
```

Comment:
Would an example of the parsed syntax make sense here (and in other complex parsers) ?